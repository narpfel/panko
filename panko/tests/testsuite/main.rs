#![feature(unqualified_local_imports)]

fn main() {
    panko_compiletest::run()
}
