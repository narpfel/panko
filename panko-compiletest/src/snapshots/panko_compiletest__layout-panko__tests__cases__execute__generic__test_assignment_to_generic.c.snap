---
source: panko/tests/testsuite/main.rs
info:
  program: panko
  args:
    - "--print=layout"
    - "--stop-after=layout"
    - panko/tests/cases/execute/generic/test_assignment_to_generic.c
---
success: true
exit_code: 0
----- stdout -----
(translation-unit
   (function-definition
      (main~0 `fn() -> int` (static main))
      (compound-statement
         (definition (value~1 `int` @0) (27 `int` @0))
         (expression
            (assign `int` @0
               (value~1 `int` @0)
               (42 `int` @0)))
         (return (value~1 `int` @0)))))

----- stderr -----
